openpype/hosts/max/api/plugin.py

```diff
@@ -246,14 +248,25 @@ def collect_instances(self):
     def update_instances(self, update_list):
         for created_inst, changes in update_list:
             instance_node = created_inst.get("instance_node")
-
             new_values = {
                 key: changes[key].new_value
                 for key in changes.changed_keys
             }
+            subset = new_values.get("subset", "")
+            if subset:
+                if instance_node != subset:
+                    node = rt.getNodeByName(instance_node)
+                    new_subset_name = new_values["subset"]
+                    if rt.getNodeByName(new_subset_name):
+                        raise CreatorError(
+                            "The subset '{}' already exists.".format(
+                                new_subset_name))
+                    created_inst["instance_node"] = new_values["subset"]
+                    node.name = created_inst["instance_node"]
+
             imprint(
-                instance_node,
-                new_values,
+                created_inst["instance_node"],
```

Comment:
This is possible only with [this](https://github.com/ynput/OpenPype/pull/5911/files#r1392270231) suggestion
```suggestion
                instance_node,
```